packages/create-novu-app/index.ts

```diff
@@ -41,6 +43,22 @@ const program = new Commander.Command(packageJson.name)
   .action((name) => {
     projectPath = name;
   })
+  .option(
+    '-k, --api-key [apiKey]',
+    `
+
+  Your Novu Development environment apiKey. Note that your novu app won't
+  work with a non development environment apiKey.
+`
+  )
+  .option(
+    '-th, --tunnel-host',
```

Comment:
I find it a bit non-standard to have a shorthand flag variant with 2 characters. What do we think of shortening to `-t`? This interface shouldn't change very frequently, if we have another `t` character we can find something else. It's better to optimize for what we have today given that this is a critical part of the onboarding flow right now.

```suggestion
    '-t, --tunnel-host',
```

Reply:
Yep, that makes sense :)